docs/contributor-guide/developer-guide/database/db-schema.md

```diff
@@ -128,28 +128,31 @@ Indexes:
     "index_persisted_durable_jobs_by_execution_time" btree (execution_time)
     "index_persisted_durable_jobs_by_success_time" btree (success_time)
 
-                                                 Table "public.programs"
-            Column            |            Type             | Collation | Nullable |               Default                
-------------------------------+-----------------------------+-----------+----------+--------------------------------------
- id                           | bigint                      |           | not null | nextval('programs_id_seq'::regclass)
- name                         | character varying           |           |          | 
- description                  | character varying           |           |          | 
- block_definitions            | jsonb                       |           | not null | 
- export_definitions           | jsonb                       |           |          | 
- legacy_localized_name        | jsonb                       |           |          | 
- legacy_localized_description | jsonb                       |           |          | 
- slug                         | character varying           |           |          | 
- localized_name               | jsonb                       |           |          | 
- localized_description        | jsonb                       |           |          | 
- external_link                | character varying           |           |          | ''::character varying
- display_mode                 | character varying           |           | not null | 'PUBLIC'::character varying
- create_time                  | timestamp without time zone |           |          | 
- last_modified_time           | timestamp without time zone |           |          | 
- status_definitions           | jsonb                       |           |          | '{"statuses": []}'::jsonb
+                                                  Table "public.programs"
```

Comment:
Line 3 says "Its current state as of #40 follows". Meaning as of https://github.com/civiform/civiform/blob/main/server/conf/evolutions/default/40.sql

So updating this as you did, funny enough, makes this documentation inconsistent. I think these docs make it clear that this isn't mean to be an up-to-date view of the database; SWEs know they must run the commands at the top in order to view that. I think closing this PR is appropriate; let me know your thoughts.

Reply:
Whoops! Thanks for your close reading of that. It still bothers me that this is out of date, but keeping it up to date seems like more trouble than it's worth. I don't think it's super clear that this is out of date with the current language though, so I'll close this PR and open another one that says that more explicitly. Thanks!